ocaml/hacl-star/EverCrypt.ml

```diff
@@ -143,12 +143,13 @@ module Hash = struct
       check_max_buffer_len (C.size msg);
       assert (C.size digest = digest_len alg);
       assert (C.disjoint digest msg);
-      everCrypt_Hash_hash (alg_definition alg) (C.ctypes_buf digest) (C.ctypes_buf msg) (C.size_uint32 msg)
+      everCrypt_Hash_Incremental_hash (alg_definition alg) (C.ctypes_buf digest) (C.ctypes_buf msg) (C.size_uint32 msg)
     let finish ~st:(alg, t) ~digest =
       assert (C.size digest = digest_len alg);
       everCrypt_Hash_Incremental_finish t (C.ctypes_buf digest)
   end
-  type t = alg * hacl_Streaming_Functor_state_s___EverCrypt_Hash_state_s____ ptr
+  (* TODO: get rid of the `alg` here by using `alg_of_state` *)
```

Comment:
According to our "Every TODO needs an issue." policy, you could reference an issue here as well. But I think it's not too important right now, so feel free to ignore.